src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -85,17 +85,22 @@ class ProtoPropertyDecorator {
 
 
     boolean isMultiple() {
-        return hasValues()
+        return valuesCount > 1
+    }
+
+
+    ProtoValue getValue() {
+        innerProtoProperty.valuesList.first()
```

Comment:
Yeah, according to the spec, specifically multi-value properties can have an "empty" value : 

```
3.6.3 Single and Multi-Value Properties
A property may be a single-value or a multi-value property.
A single-value property, if it exists, must have a value. There is no such thing as a null value. A multi-value property can have zero or more values. Again there is no such thing as a null value, however a multi-value property can be empty, just as an array can be empty.
```